cmd/scollector/collectors/fastly.go

```diff
@@ -67,6 +68,7 @@ const (
 	fastlyScheduledMaintDesc       = "The number of currently scheduled maintenances. Does not include maintenance that is current active"
 	fastlyActiveScheduledMaintDesc = "The number of currently scheduled maintenances currently in progress. Includes the 'in_progress' and 'verifying'"
 	fastlyActiveIncidentDesc       = "The number of currently active incidents. Includes the 'investingating', 'identified', and 'monitoring' states."
+	fastlyMaxDuration              = 3 * time.Minute
```

Comment:
Do we know that we need this timeout?

In main.go we set the defaultclient timeouts to for http and collect to one minute.

Reply:
Will change it to one minute. Note that if c.GetSummary() was called several times or if it was doing several http requests, the whole function would have a global 1 minute boundary. 1 minute makes sense here since this check is launched every minute.